test/zdtm/static/sock_ip_opts00.c

```diff
@@ -26,6 +26,7 @@ struct sk_opt {
 struct sk_opt sk_opts_v4[] = {
 	{ SOL_IP, IP_FREEBIND, IP_OPT_VAL },
 	{ SOL_IP, IP_PKTINFO, IP_OPT_VAL },
+	{ SOL_IP, IP_TTL, IP_OPT_VAL },
```

Comment:
`val` should be a value between 1 and 255, not `IP_OPT_VAL`.
You can run the test locally with `sudo python3 zdtm.py run -t zdtm/static/sock_ip_opts01`

Reply:
I have ran the tests locally. Changed the value to 32 .